wallet-webapps-common/src/main/webapp/vue-app/components/SendTokensForm.vue

```diff
@@ -550,17 +550,17 @@ export default {
             }
             if (savedTransaction) {
               this.finalizeSendTransaction(savedTransaction, this.contractDetails, this.notificationId);
+              this.close();
```

Comment:
you may need to centralize this behavior by adding it inside `this.finalizeSendTransaction` knowing that you have there `this.$emit('close')`

Reply:
I added an event listener` this.$on('close',this.close())`